draft-pignataro-eimpact-icmp.xml

```diff
@@ -458,7 +458,48 @@ The "Year" field must contain the year in which the certificate was issued, or 0
 
     <section title="Sample Use Case">
       <t>
-        This section explains a typical use case of the extensions.
+        The ICMP extension defined in this memo can be used to get sustainability metrics, either via a
+        "transactional" or a "scheduled automated" fashion, inside one administrative domain.
+        Mentioned in this section is a sample "transactional" use case of how the metrics will be
+        displayed to the user. As traceroute or ping utilities are based on ICMP, we will use
+        traceroute as an example.
+      </t>
+      <t>  
+        The flavour of traceroute that has been shown here, does support the ICMP extensions and is
+        capable of deliverintg the metrics in the extensions to the end user along with the nodes that 
+        the original datagram visited on it's way to the destination. And, it also is backwards-compatible,
+        i.e., it can also work well with those nodes which doesn't support ICMP extensions.
+      </t>
+      <t>
+        Host A executes a traceroute to 192.168.2.1 and the <xref target="use-case-1" /> shows how the user
+        of Host A will be presented with the metrics:
+        <figure anchor="use-case-1" title="Sustainability Metrics infused Traceroute">
+          <artwork xml:space="preserve" >
+> traceroute 192.168.2.1
+Tracing route to 192.168.2.1 over a maximum of 30 hops
+
+  1     3 ms     1 ms     2 ms  192.168.2.120
```

Comment:
```suggestion
  1     3 ms     1 ms     2 ms  192.168.2.125
```
It really doesn't matter, but I find the sequence 120, 124, 123, 122 distracting.